docker/maintenance/cronjob-template.yaml

```diff
@@ -15,10 +15,16 @@ spec:
           - name: php
             image: artifactory.wikia-inc.com/sus/mediawiki-prod-php:${label}
             env:
-             - name: SERVER_ID
-               value: "${server_id}"
-             - name: LOG_STDOUT_ONLY
-               value: "yes"
+              - name: KUBERNETES_DEPLOYMENT_NAME
+                value: mediawiki-prod
```

Comment:
Is there a reason to not use `mw-cj-${name}` here and avoid calling production pods and slowing them down?

Reply:
Well, there's no nginx here :)

Reply:
Is it the same in the old setup? I just wonder if we won't overload the production pods with heavy queries which could be confined to the cronjob pods. Would it be worth to add the nginx here?

Reply:
I'd actually move these requests to `mediawiki-tasks`, so that the offline traffic will be separated from end-user requests. Good point!